src/main/java/com/uber/cadence/client/WorkflowClientOptions.java

```diff
@@ -175,4 +200,52 @@ public String getIdentity() {
   public List<ContextPropagator> getContextPropagators() {
     return contextPropagators;
   }
+
+  public QueryRejectCondition getQueryRejectCondition() {
+    return queryRejectCondition;
+  }
+
+  @Override
+  public String toString() {
+    return "WorkflowClientOptions{"
+        + "domain='"
+        + domain
+        + '\''
+        + ", dataConverter="
+        + dataConverter
+        + ", interceptors="
+        + Arrays.toString(interceptors)
+        + ", identity='"
+        + identity
+        + '\''
+        + ", contextPropagators="
+        + contextPropagators
+        + ", queryRejectCondition="
+        + queryRejectCondition
+        + '}';
+  }
+
+  @Override
+  public boolean equals(Object o) {
```

Comment:
It looks that `equals` and `hashCode` are not used anywhere. Do we really need those methods?